css/style.css

```diff
@@ -0,0 +1,155 @@
+.page{
+	background: url("../images/backgr.jpg") gray;
+	-moz-background-size: 100%; /* Firefox 3.6+ */
+    -webkit-background-size: 100%; /* Safari 3.1+ и Chrome 4.0+ */
+    -o-background-size: 100%; /* Opera 9.6+ */
+	background-size: 100%;
+}
+
+.content{
+	position: relative;
+	margin: 10% auto;
+	padding: 15px;
+	width: 700px;
+	height: 50%;
+	background-color: #111a28;
+	border-radius: 10px;
+}
+
+.content__logo{
+	margin-top: 20px;
+	margin-left: 15px;
+}
+
+.product__photo{
+	display: inline-block;
+	width: 65%;
+}
+
+.photo{
+	margin: 50px 10px;
+}
+
+.photo__colorSelector{
+	margin: auto 30%;
+}
+
+.photo__colorSelector__button{
```

Comment:
you cannot use cammel case style in bame
and have 2 elements
you should have smth like photo__button